root/utility/edit.js

```diff
@@ -55,19 +54,36 @@ export function getEditStatusName(edit: EditT) {
 export function getEditStatusDescription(edit: EditT) {
   switch (edit.status) {
     case EDIT_STATUS_OPEN:
-      return l('This edit is open and awaiting votes before it can be applied.');
+      return l(
+        'This edit is open and awaiting votes before it can be applied.',
+      );
     case EDIT_STATUS_APPLIED:
       return l('This edit has been successfully applied.');
     case EDIT_STATUS_FAILEDVOTE:
-      return l('This edit failed because there were insufficient "yes" votes.');
+      return l(
+        'This edit failed because there were insufficient "yes" votes.',
+      );
```

Comment:
FWIW, formatting changes like these make it a lot harder to review wording changes, as it makes it a lot harder to tell when it’s just a formatting change and when wording has actually changed as well.

Reply:
That's why it's on a different commit :) d3a8d3f4bc48e9be77382961879efb683db4dd64 and bcc306703d43645af5ade7c9a2a9486afc23c999 are the only ones which affect display, unless I messed up somewhere.